dotcom-rendering/src/web/server/articleTemplate.ts

```diff
@@ -275,6 +277,7 @@ https://workforus.theguardian.com/careers/product-engineering/
 					${initTwitter}
 				</script>
 
+				${recipeMarkup}
```

Comment:
I think it would be easier to read this file and follow the conventions here if we pass the raw json as the string parameter and set the script tags explicitly here in `articleTemplate`.